invenio_vocabularies/contrib/funders/mappings/v7/funders/funder-v1.0.0.json

```diff
@@ -0,0 +1,77 @@
+{
+  "mappings": {
+    "dynamic": "strict",
+    "properties": {
+      "$schema": {
+        "type": "keyword",
+        "index": "false"
+      },
+      "created": {
+        "type": "date"
+      },
+      "updated": {
+        "type": "date"
+      },
+      "uuid": {
+        "type": "keyword"
+      },
+      "version_id": {
+        "type": "integer"
+      },
+      "id": {
+        "type": "keyword"
+      },
+      "identifiers" : {
+        "properties": {
+          "identifier" : {
+            "type" : "keyword"
+          },
+          "scheme" : {
+            "type" : "keyword"
+          }
+        }
+      },
+      "name_sort": {
+        "type": "keyword"
+      },
+      "name": {
+        "type": "text",
+        "copy_to": "name_sort",
+        "fields": {
+          "suggest": {
+            "type": "search_as_you_type"
+          }
+        }
+      },
+      "country": {
+        "type": "text"
+      },
+      "pid": {
+        "type": "object",
+        "properties": {
+          "pk": {
+            "type": "integer"
+          },
+          "pid_type": {
+            "type": "keyword"
+          },
+          "obj_type": {
+            "type": "keyword"
+          },
+          "status": {
+            "type": "keyword"
+          }
+        }
+      },
+      "title": {
+        "type": "object",
+        "dynamic": true,
+        "properties": {
+          "en": {
+            "type": "text"
```

Comment:
moderate: missing `search_as_you_type` (only for v7). See [affiliations](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/contrib/affiliations/mappings/v7/affiliations/affiliation-v1.0.0.json#L76) as example.